src/API/AbstractPluginActions.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace CF\API;
+
+use CF\Integration\DataStoreInterface;
+use CF\Integration\DefaultIntegration;
+use CF\WordPress\Constants\Exceptions\PageRuleLimitException;
+use CF\WordPress\Constants\Exceptions\ZoneSettingFailException;
+use CF\WordPress\Constants\Plans;
+
+abstract class AbstractPluginActions
```

Comment:
No, I think #2 is the best route.  In my opinion its self documenting.  If someone sees login doesn't work, they'll look at the route, the plugin actions file and see it was overwritten.  The only change I would make `login()` return an error message about being unimplemented etc.